docs/tutorials/connecting-to-ev3dev-with-ssh.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Connecting to Ev3dev Using SSH
```

Comment:
This is the "new grammar" as I have taken to calling it :laughing: The issue is that we have a tradeoff between not capitalizing it (even though it is title case) and capitalizing it weirdly as you pointed out. So, we're looking at this:

### Doing Stuff with ev3dev
### Doing Stuff with Ev3dev

Both look wrong. One is more grammatical, the other may look better. We opted to standardize on the latter option, but honestly I don't have much of a preference either way.